src/components/mdCheckbox/mdCheckbox.scss

```diff
@@ -19,6 +19,7 @@ $checkbox-touch-size: 48px;
 
     &:focus {
       outline: none;
+      box-shadow: 0 0 0 2px rgba(#000, .14);
```

Comment:
I think that instead of box-shadow we should highlight the md-ripple element inside. Do you think that you can change that?